sdks/python/apache_beam/runners/worker/bundle_processor.py

```diff
@@ -381,8 +381,12 @@ def set(self, ts):
         windowed_value.WindowedValue(
             (self._key, dict(timestamp=ts)), ts, (self._window,)))
 
-  def clear(self, timestamp):
-    self._receiver.receive((self._key, dict(clear=True)))
+  def clear(self):
+    dummy_millis = int(common_urns.constants.MAX_TIMESTAMP_MILLIS.constant) + 1
```

Comment:
Did you mean to use GlobalWindow.MAX_TIMESTAMP here?

I think this should be just MAX_TIMESTAMP (see above).

Reply:
No, it refers to https://github.com/apache/beam/blob/7d6b7b89f5832d5d375059f9c1980181b443835b/model/pipeline/src/main/proto/beam_runner_api.proto#L40

Reply:
Wouldn't this cause an overflow then? We cannot go past the maximum timestamp.

Reply:
Long.MAX_VALUE is `9223372036854775807`, which is different from `MAX_TIMESTAMP_MILLIS`. The resulting timer timestamp is `Timer{timestamp=294247-01-10T04:00:54.776Z, payload=[B@5aec2a63}`. (The code in this PR is covered by a test, if there was an overflow the test would fail.)

Reply:
Ok, max timestamp is then `Long.MAX_VALUE / 1000`. Seems safe to add 1 to it then.